src/stories/containers/TransparencyReport/components/AccountsSnapshot/api/queries.ts

```diff
@@ -52,7 +80,57 @@ export const fetchAccountsSnapshot = async (ownerType: string, ownerId: string):
     snapshots: Snapshots[];
   }>(GRAPHQL_ENDPOINT, query, filter);
 
-  console.log(res);
-
   return res.snapshots;
 };
+
+interface CoreUnitResponse {
+  coreUnits: {
+    shortCode: string;
+  }[];
+}
+
+interface TeamResponse {
+  teams: {
+    shortCode: string;
+  }[];
+}
+
+const getCoreUnitShortCode = async (ownerId: string): Promise<string> => {
+  const { query, filter } = coreUnitShortCodeQuery(ownerId);
+  const res = await request<CoreUnitResponse>(GRAPHQL_ENDPOINT, query, filter);
+  if (res && res.coreUnits && res.coreUnits[0] && res.coreUnits[0].shortCode) {
```

Comment:
In this case, it's not a better option to use. operator?

Reply:
You're right, changing now...